### PR TITLE
Added more logging to IMU failure

### LINKED ISSUE #39: Add relevant logging at IMU start up
Currently the IMU setup will report some data if the setup fails, but we would like a little more information both on failure on success. We should log both accel and gyro values regardless of result.

### PR DESCRIPTION
**Pull request by ABenoit01**
## Description of Problem
Added more logging for gyro values in IMU failure.
closes #39 

## Testing
- compiled code with no errors or warnings

### CODE REVIEW

File: main/gy521_imu.h [hunk 1]
```diff
@@ -79,5 +79,17 @@ void setupIMU(){
     Serial1.print("Z Acceleration: ");
     Serial1.println(sample.accel.z);
     delay(1000);
-  }
+    imuReading imuSample = getIMU();
```
Comment (JonathanSafer) [inline main/gy521_imu.h:82]: We already have an imuReading on line 67, so you can remove this line and use `sample` in all of the places that you used `imuSample`

File: main/gy521_imu.h [hunk 2]
```diff
@@ -79,5 +79,17 @@ void setupIMU(){
...
+    Serial1.print("X Gyro");
```
Comment (JonathanSafer) [inline main/gy521_imu.h:83]: `print` doesn't add any formatting or spaces, so in this current configuration it'll come out as `X Gyro1.234`. If you want `X Gyro: 1.234` you'll have to change this line to `"X Gyro: "`. Same with the rest.

File: main/gy521_imu.h [hunk 3]
```diff
@@ -79,5 +79,17 @@ void setupIMU(){
...
+    Serial1.println(imuSample.gyro.x);
+    delay(1000);
+    Serial1.print("Y Gyro");
+    Serial1.println(imuSample.gyro.y);
+    delay(1000);
+    Serial1.print("Z Gyro");
+    Serial1.println(imuSample.gyro.z);
+    delay(1000);
+    
+    Serial1.print("The log worked"); //this is just here for testing purposes
```
Comment (JonathanSafer) [inline main/gy521_imu.h:93]: This line isn't necessary, it'll be clear that the log worked (or didn't)

File: main/gy521_imu.h [hunk 4]
```diff
@@ -79,5 +79,17 @@ void setupIMU(){
...
-  }
+    imuReading imuSample = getIMU();
+    Serial1.print("X Gyro");
+    Serial1.println(imuSample.gyro.x);
+    delay(1000);
+    Serial1.print("Y Gyro");
+    Serial1.println(imuSample.gyro.y);
+    delay(1000);
+    Serial1.print("Z Gyro");
+    Serial1.println(imuSample.gyro.z);
```
Comment (JonathanSafer) [inline main/gy521_imu.h:90]: Could you also add logs for all of the accel values?

**Review by JonathanSafer — CHANGES_REQUESTED** [review]
Looks good, just a few minor nitpicks

**Comment by JonathanSafer** [comment]
You can actually just take all the lines out of the `else` section, since those should run regardless of IMU outcome (except for the "IMU failed" one)

**Comment by JonathanSafer** [comment]
Also in the Description you wrote that this is logging for an IMU failure, but it's actually logging regardless of success or failure

File: main/gy521_imu.h (hunk 4 above)
Comment (ABenoit01) [reply main/gy521_imu.h:90]: They already are logs for accel values.

File: main/gy521_imu.h (hunk 2 above)
Comment (ABenoit01) [reply main/gy521_imu.h:83]: Oh yeah, had some trouble with updating the branch so I had to default to old code that I had written which didn't have the space

**Review by ABenoit01 — COMMENTED** [review]
Thank you for the feedback!

**Comment by ABenoit01** [comment]
I did something wrong so now this pull request is #118

**Pull request closed without merging**